File: Modul 7/Soal 1/PRAK601-2210817220018-Piola Evania.py
for i in range(3):   
    b, k = map(int, input().split())
    m = []
    x = 0
    isi = list(map(int, input().split()))
    for i in range(b):
        m.append(isi[x : x + k])
        x = x + k
    for b in m:
        for isi in b:
            print(isi, end=' ')
        print()
    print("\n")

File: Modul 7/Soal 1/PRAK601-2210817220018-Piola Evania.c
#include <stdio.h>

int main(void)
{
    for(int i = 1; i < 4; i++){
       int b, k, isi;
       scanf("%d %d", &b, &k);
       int m[b][k];
       for(int i = 0; i < b; i++){
        for(int j = 0; j < k; j++){
            scanf("%d", &isi);
            m[i][j] = isi;
        }
       }
       printf("\n");
       for(int i = 0; i < b; i++){
        for(int j = 0; j < k; j++){
            printf("%d ", m[i][j]);
        }
        printf("\n");
       }
       printf("\n");
    }
}
